### PR TITLE
typos corrected and added some explanations

### PR DESCRIPTION
**Pull request by hkey0**
Minor typos corrected and explanations added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,9 +18,9 @@ Sinker is built by [Paradigm](https://www.paradigm.xyz/), and is licensed under
 
 In Postgres, you might have a normalized schema like this:
 
-- A Student and a Teacher refer to a Person
+- A Student and a Teacher refers to a Person
```
Comment (loren) [inline README.md:21]: this is correct as is

File: README.md [hunk 2]
```diff
@@ -18,9 +18,9 @@ Sinker is built by [Paradigm](https://www.paradigm.xyz/), and is licensed under
...
 - A Course is taught by a Teacher
-- Students have and belong to many Courses through the Enrollment join table
+- Students have taken and belong to many Courses through the Enrollment join table
```
Comment (loren) [inline README.md:23]: I'd prefer to keep this language as HABTM is a standard way to indicate a many-to-many relationship.

File: README.md [hunk 3]
```diff
@@ -78,7 +78,7 @@ Sinker creates triggers on the Postgres tables that you want to synchronize (e.g
 above). When a row is inserted, updated, or deleted in any of these tables, the trigger schedules the materialized view
 to be refreshed at the next interval.
 
-The changes to the materialized view are sent to a logical replication slot. Sinker reads from this slot and indexes the
+The changes into the materialized view are sent to a logical replication slot. Sinker reads from this slot and indexes the
```
Comment (loren) [inline README.md:81]: I think "changes to" is correct as is.

File: README.md [hunk 4]
```diff
@@ -331,7 +331,7 @@ Spin up Postgres and Elasticsearch:
 docker-compose --env-file=.env.test up -d
 ```
 
-Run tests:
+Run tests (with -s option to allow more verbose output):
```
Comment (loren) [inline README.md:334]: 👍

**Review by loren — CHANGES_REQUESTED** [review]

**Comment by hkey0** [comment]
Thank you for your attention. I have made the requested corrections. Thanks again :)

**Review by loren — APPROVED** [review]
thank you for contributing!

**Pull request merged**